apps/web/vibes/soul/sections/cart/index.tsx

```diff
@@ -53,71 +46,62 @@ interface CartState<LineItem extends CartLineItem> {
   lastResult: SubmissionResult | null;
 }
 
+export interface Cart<LineItem extends CartLineItem> {
+  lineItems: LineItem[];
+  summaryLineItems: CartSummaryLineItem[];
+  total: string;
+}
+
 interface CartProps<LineItem extends CartLineItem> {
   title?: string;
-  lineItems: LineItem[] | Promise<LineItem[]>;
-  summary: CartSummary;
-  emptyState: CartEmptyState;
+  summaryTitle?: string;
+  emptyState?: CartEmptyState;
   lineItemAction: Action<CartState<LineItem>, FormData>;
   checkoutAction: Action<SubmissionResult | null, FormData>;
+  checkoutLabel?: string;
+  summaryTotalLabel?: string;
```

Comment:
What do you think if we had a `summary` prop and it had to fields, `items` and `subtotal` and `items` was a `CartSummaryItem[]` and `subtotal` was `CartSummaryItem` but we render it with a different visual style?

Reply:
`subtotal` seems to be rendered as just a regular summary item though, why separate it? Are you maybe getting it confused with `total` which does need to be separated? I went back and forth on this for a while and figured that grouping things into `summary` was unnecessary. The concept of a `total` price is represented in different ways on different carts and may not necessarily be in the summary section. Nesting things inside of `summary` makes an assumption about the UI which I felt would potentially cause awkwardness down the road in other VIBES.

Reply:
I thought "total" was only shown in the checkout view and that the cart only had a subtotal, but yeah, I meant the item that was itself styled a bit differently than the others. I wasn't thinking of the summary section when I read the summary prop and was thinking more about the summary of the cart concept itself, not any particular UI. It just feels a bit inconsistent what we've been nesting and what we haven't been nesting. It's not a big deal, though